src/style.css

```diff
@@ -0,0 +1,112 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.search-form {
+  background-color: #fff;
+  border-radius: 4px;
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  margin: 20px 0;
+}
+
+.form-small {
+  box-sizing: border-box;
+  width: 100%;
+  height: 42px;
+}
+
+.searchIcon {
+  position: relative;
+  width: 100%;
+}
+
+.searchIcon::after {
+  position: absolute;
+  content: "";
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 0;
+  left: 26px;
+}
+
+.searchIcon-Small::after {
+  width: 11px;
+  height: 11px;
+  top: 5px;
+  left: 13px;
+}
+
+.textSearch::placeholder {
+  font-family: inherit;
+  font-size: 16px;
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.textSearch {
+  font-family: Avenir, sans-serif;
```

Comment:
you need just one time put needed font to the HTML selector and then inherit it where needed